sdk/include/psdk/ntgdi.h

```diff
@@ -2457,7 +2457,7 @@ BOOL
 APIENTRY
 NtGdiGetTextExtent(
     _In_ HDC hdc,
-    _In_reads_(cwc) LPWSTR lpwsz,
+    _In_reads_(cwc) LPCWSTR lpwsz,
```

Comment:
```suggestion
    _In_reads_(cwc) PCWSTR lpwsz,
```
If the string isn't actually NUL-terminated, I would suggest `PCWCH` instead.

Reply:
```suggestion
    _In_reads_(cwc) PCWCH lpwsz,
```